paasta_tools/api/api_docs/swagger.json

```diff
@@ -330,8 +330,8 @@
                     },
                     {
                         "in": "query",
-                        "description": "Omit Smartstack information",
-                        "name": "omit_smartstack",
+                        "description": "Include Smartstack information",
+                        "name": "include_smartstack",
                         "required": false,
                         "type": "boolean"
                     },
```

Comment:
Yeah, I can change `omit_mesos` too. Another option would be keeping it as the current status until we remove mesos related code. 
I agree this might break paasta status for a short period because of the different issue (Although I think it's for different reason, which client tries to get the params that's not provided by API server). Doing the two-phase change seems not worth the effort. I would like to ship this with @solarkennedy's change for removing chronos together, so we only need to break it once.